examples/babyai/config/tensorzero.toml

```diff
@@ -9,73 +9,57 @@ routing = ["openai"]
 type = "openai"
 model_name = "gpt-4o-mini-2024-07-18"
 
-[models."ft:gpt-4o-mini-2024-07-18:tensorzero::ApNUdVRd"]
-routing = [ "openai",]
+[models."ft:gpt-4o-mini-2024-07-18:tensorzero::Arsyj2TT"]
+routing = [ "ft:gpt-4o-mini-2024-07-18:tensorzero::Arsyj2TT" ]
 
-[models."ft:gpt-4o-mini-2024-07-18:tensorzero::ApNUdVRd".providers.openai]
+[models."ft:gpt-4o-mini-2024-07-18:tensorzero::Arsyj2TT".providers."ft:gpt-4o-mini-2024-07-18:tensorzero::Arsyj2TT"]
 type = "openai"
-model_name = "ft:gpt-4o-mini-2024-07-18:tensorzero::ApNUdVRd"
+model_name = "ft:gpt-4o-mini-2024-07-18:tensorzero::Arsyj2TT"
```

Comment:
we should remove this from the example and just ask people to run a fine-tuning recipe and follow the instructions.